src/highlight/highlightUtil.ts

```diff
@@ -5,7 +5,7 @@ export function isHighlight(annotation: Annotation): annotation is AnnotationHig
     return annotation?.target?.type === Type.highlight;
 }
 
-export function isValidHighlight({ target }: AnnotationHighlight): boolean {
+export function isValidHighlight({ target }: Pick<AnnotationHighlight, 'target'>): boolean {
```

Comment:
Is the idea to allow an incomplete AnnotationHighlight to be passed in? If so, this seems really specific to HighlightList. Should it be encapsulated there?